lib/matplotlib/__init__.py

```diff
@@ -809,6 +808,19 @@ def copy(self):
             rccopy._set(k, self._get(k))
         return rccopy
 
+    def _load_path_effects(self):
+        """defers loading of patheffects to avoid circular imports"""
+        import matplotlib.patheffects as path_effects
+
+        path_effects_list = self._get('path.effects').copy()
+        for k, v in self.find_all('path.effects.').items():
+            if v is None:
+                continue
+            path_effect_name = k.split('.')[-1]
+            path_effect_function = getattr(path_effects, path_effect_name)
+            path_effects_list.append(path_effect_function(**v))
+        return path_effects_list
```

Comment:
I suspect this makes the order of entries in the style file matter (because if you have multiple patheffects they will be applied in order, and you can e.g. swap the order of the two patheffects on txt.arrow_patch in patheffect_demo.py to see that the order of patheffects is indeed important).  Technically we don't guarantee that matplotlibrc order doesn't matter, but that still seems a bit weird.

I guess the safer way would be to take a single list, e.g. something like `path.effects: [{"__name__": "withStroke", "kw1": val1, "kw2": val2, ...}]`.

Reply:
Technically you may also want to apply the same path effect twice with different kwargs (a realistic use case could be wanting two different tickedstrokes on both sides of a path), which a dict won't support.